setup.py

```diff
@@ -30,7 +30,10 @@
         'tabulate'
     ],
     url='https://github.com/dataxu/github-pr',
-    version='1.1.1',
+    version='2.0.0',
     scripts=['github_pr.py'],
     keywords=['github'],
+    dependency_links=[
+        "git+https://github.com/dataxu/PyGithub#egg=PyGithub-1.26.1"
```

Comment:
@cchristous i think the other reason we did this, was that my PR for getting the 'since' added to the params for options when returning comments was merged, but they didnt create a release. So we don't have a way other than us creating a release on our fork (which we did) to get that change, which is needed here.